doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "NASYS LoRaWAN® Outdoor Gateway"
+description: ""
+weight: 1
+---
+
+NASYS LoRaWAN® Outdoor Gateway is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official product page][1]. This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. LoRaWAN® Outdoor Gateway connected to the Internet (or your local network) via Ethernet.
```

Comment:
Include NASYS here as well

Reply:
Also no title case for internet, ethernet, gateways, etc

Reply:
Fixed